hack/import-restrictions.yaml

```diff
@@ -35,6 +35,7 @@
 - baseImportPath: "./vendor/k8s.io/api/"
   allowedImports:
   - k8s.io/api
+  - k8s.io/apiserver/pkg/apis/audit
```

Comment:
this is not allowed, leads to an import circle.

Reply:
It relies on the `Policy` type in that is defined in `audit/v1`. We could duplicate that object if necessary, but there is nothing importing the api package from there.

Reply:
Probably you don't need that exception, do you?

Reply:
It depends on how we want to solve it. We could duplicate Policy from `audit/v1` and provide translation methods, or just allow that import as a special use case. Not sure whats more desirable here

Reply:
We cannot allow this dependency. We have to move the type. We can though reference the k8s.io/api policy type from k8s.io/apiserver to avoid duplication.